src/components/Documents/DocumentTable.jsx

```diff
@@ -28,7 +28,7 @@ import { EmptyListNotification, LoadingAnimation } from '../Notification';
  * @param {documentTableProps} Props - Props for DocumentTable component
  * @returns {React.JSX.Element} The DocumentTable component
  */
-const DocumentTable = ({ handleAclPermissionsModal }) => {
+const DocumentTable = ({ handleAclPermissionsModal, handleSelectDeleteDoc }) => {
```

Comment:
Think we could expand the JSDoc for this component props to include `handleSelectDeleteDoc`

Reply:
I think using `Function` is fine. Although I wish it can be more explicit like with `handleAclPermissionsModal`.

Example with `handleAclPermissionsModal`:
<img width="1440" alt="Screen Shot 2023-09-14 at 23 01 43" src="https://github.com/codeforpdx/PASS/assets/14917816/1e9342dd-d633-472f-b57e-aa1c9856ed90">

Example with generic `Function`:
<img width="1440" alt="Screen Shot 2023-09-14 at 23 01 53" src="https://github.com/codeforpdx/PASS/assets/14917816/875b2529-7e90-4558-92df-c9f081320dd0">